src/avrdude.1

```diff
@@ -2074,13 +2074,15 @@ Show help menu and exit.
 .El
 .El
 .Ss Programmers accepting exitspec parameter
-Currently, only the flip2, linuxspi and old school parallel port programmers
-such as stk200 and dapa support exitspec parameter. These lets the user
-decide in which state the programmer pins after ended programming session.
+Currently, only the flip2, linuxspi, linuxgpio, raspberry_pi_gpio and old school parallel
+port programmers such as stk200 and dapa support exitspec parameter.
+These lets the user decide in which state the programmer pins after ended programming session.
```

Comment:
Pre-existing typo: These let the user (not lets)

Reply:
Thanks! I'll fix it right away. These are the traps a native Norwegian usually falls into, due to how Norwegian grammar works.